<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>configOptions</class>
 <widget class="QDialog" name="configOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>General Options</string>
    </attribute>
    <widget class="QCheckBox" name="cbAutoOpenMountPoint">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbReUseMountPoint">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAutoCheckForUpdates">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbStartMinimized">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSelectLanguage">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>331</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Language</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Open Mount Point</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Re Use Mount Point</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Check For Updates</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>160</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Minimized</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAutoMountAtStartUp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>200</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Mount Favorites Volumes At Startup</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAutoMountWhenAvailable">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>431</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>240</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Auto Mount Favorites Volumes When Available</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbShowMountDialogWhenAutoMounting">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>280</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Mount Dialog When Auto Mounting</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbAllowExternalToolsToReadPasswords">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>320</y>
       <width>21</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>320</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Allow External Tools To Get Passwords From Wallets</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="cbShowFavoriteListOnContextMenu">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Favorite List On Context Menu(Restart Required)</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="HiDPI_Settings">
    <attribute name="title">
     <string>HiDPI Settings</string>
    </attribute>
    <widget class="QCheckBox" name="cbHiDPI">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditHiDPI">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>241</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Scale Factor
(Enter A Decimal Number)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enable HiDPI Scalling</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>551</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>511</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>SiriKali Will Have To Be Restarted For These Changes To Take Effect.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="manageKeys">
    <attribute name="title">
     <string>Manage Keys In Wallets</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>491</width>
       <height>321</height>
      </rect>
     </property>
     <property name="title">
      <string>Set Default Wallet To Retrieve Volume Keys</string>
     </property>
     <widget class="QRadioButton" name="rbInternalWallet">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>90</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Internal Wallet</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbLibSecret">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;Libsecret</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbKWallet">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>210</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;KWallet</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbMacOSKeyChain">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>270</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;MACOS Keychain</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbNone">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&amp;None</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pbChangeWalletPassword">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>431</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Change Internal Wallet Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pbKeyStorage">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>360</y>
       <width>291</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Key Storage</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="setExecutables">
    <attribute name="title">
     <string>External Commands</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set File Manager</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditExecutableKeySource">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMountPointPrefix">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditFileManager">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>90</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Mount Point Prefix</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMountPointPrefix">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>30</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>120</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set External Plugin Executable</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Command To Run After Mount</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditAfterMountCommand">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>210</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditBeforesUnMount">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>270</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Command To Run Before UnMount</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPostMountCommand">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>210</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbPreUnMountCommand">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>270</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>300</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Command To Run Periodically</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRunPeriodically">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>391</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunPeriodicallyCommand">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>330</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditRunPeriodicallyInterval">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>360</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Interval In Minutes Periodic Command Will Run</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetFileManager">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>90</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetExternalCommand">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>150</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbExternalExecutableDefault">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>150</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbSetFileManagerToDefault">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>90</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbMountPointPrefixDefault">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>30</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunPostMountCommandDefault">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunPreUnMountCommandDefault">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>270</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pbRunCommandPeriodicallyDefault">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>330</y>
       <width>31</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Set To Default</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>440</y>
     <width>131</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cbAutoOpenMountPoint</tabstop>
  <tabstop>cbReUseMountPoint</tabstop>
  <tabstop>cbAutoCheckForUpdates</tabstop>
  <tabstop>cbStartMinimized</tabstop>
  <tabstop>cbAutoMountAtStartUp</tabstop>
  <tabstop>cbAutoMountWhenAvailable</tabstop>
  <tabstop>cbShowMountDialogWhenAutoMounting</tabstop>
  <tabstop>cbAllowExternalToolsToReadPasswords</tabstop>
  <tabstop>pbSelectLanguage</tabstop>
  <tabstop>rbNone</tabstop>
  <tabstop>rbInternalWallet</tabstop>
  <tabstop>rbLibSecret</tabstop>
  <tabstop>rbKWallet</tabstop>
  <tabstop>rbMacOSKeyChain</tabstop>
  <tabstop>pbChangeWalletPassword</tabstop>
  <tabstop>pbKeyStorage</tabstop>
  <tabstop>lineEditMountPointPrefix</tabstop>
  <tabstop>pbMountPointPrefix</tabstop>
  <tabstop>pbMountPointPrefixDefault</tabstop>
  <tabstop>lineEditFileManager</tabstop>
  <tabstop>pbSetFileManager</tabstop>
  <tabstop>pbSetFileManagerToDefault</tabstop>
  <tabstop>lineEditExecutableKeySource</tabstop>
  <tabstop>pbSetExternalCommand</tabstop>
  <tabstop>pbExternalExecutableDefault</tabstop>
  <tabstop>lineEditAfterMountCommand</tabstop>
  <tabstop>pbPostMountCommand</tabstop>
  <tabstop>pbRunPostMountCommandDefault</tabstop>
  <tabstop>lineEditBeforesUnMount</tabstop>
  <tabstop>pbPreUnMountCommand</tabstop>
  <tabstop>pbRunPreUnMountCommandDefault</tabstop>
  <tabstop>lineEditRunPeriodically</tabstop>
  <tabstop>pbRunPeriodicallyCommand</tabstop>
  <tabstop>pbRunCommandPeriodicallyDefault</tabstop>
  <tabstop>lineEditRunPeriodicallyInterval</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
